site/src/pages/AuditPage/AuditPageView.tsx

```diff
@@ -72,53 +78,88 @@ export const AuditPageView: FC<AuditPageViewProps> = ({
         <PageHeaderSubtitle>{Language.subtitle}</PageHeaderSubtitle>
       </PageHeader>
 
-      <SearchBarWithFilter
-        docs="https://coder.com/docs/coder-oss/latest/admin/audit-logs#filtering-logs"
-        filter={filter}
-        onFilter={onFilter}
-        presetFilters={presetFilters}
-      />
+      <ChooseOne>
```

Comment:
I would break these two outcomes out into separate files: one for the case where audit logs should be hidden, and one for their visibility.

Reply:
What benefits do you see on that?

Reply:
But no strong opinion, I know a lot of people don't like it 😄

Reply:
I hear some of your concerns here for sure. I wonder if we could leverage a better directory structure to help keep composable pieces of code that are related close together, but still have more readable files.

Reply:
Sure thing!

Reply:
Personally I also like shorter files because I feel less likely to overlook something. I think the concern about stuff that's reused is important but that we can put everything related in one folder, like Kira says. I won't be heartbroken either way though.

Reply:
Lets go with shorter files!

Reply:
Another thought, is it ok just to move the Paywall? I would avoid to have one more level of prop drilling

Reply:
Ya, definitely!